packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -9,10 +9,10 @@
 
 .edit-post-visual-editor .block-editor-writing-flow__click-redirect {
 	// Collapse to minimum height of 50px, to fully occupy editor bottom pad.
-	height: 50px;
+	height: 240px;
 	width: 100%;
 	// Offset for: Visual editor padding, block (default appender) margin.
-	margin: #{ -1 * $block-spacing } auto -50px;
+	margin: #{ -1 * $block-spacing } auto -240px;
```

Comment:
The `-50px` looks like it's present because the parent of the 'click-redirect' element (`.edit-post-visual-editor`) has a 50px padding at the top and bottom:
https://github.com/WordPress/gutenberg/blob/1385adf98197a4f12b1490c87a3487ffad2c88a3/packages/edit-post/src/components/visual-editor/style.scss#L3

The `-50px` margin that was previously specified essentially counteracts that and pulls the 'click-redirect' element down to the bottom of the screen. Not sure why it continues to work when it's `-240px`, but `-50px` is technically the correct value.

TLDR, sorry to ask you to change this back again, but it looks like it should stay at 50px.